examples/code_general_impots/Section_I_Dispositions_generales.catala

```diff
@@ -0,0 +1,40 @@
+@@Code général des impôts@@
+@@Livre premier : Assiette et liquidation de l'impôt@@+
+@@Première Partie : Impôts d'État@@++
+@@Titre premier : Impôts directs et taxes assimilées@@+++
+@@Chapitre premier : Impôt sur le revenu@@++++
+@@Section I : Dispositions générales@@+++++
+@@0I : Définition du revenu net global@@++++++
+@Article 1 A En savoir plus sur cet article...@
+Créé par Loi 2005-1719 2005-12-30 art. 74 I Finances pour 2006 JORF 31 décembre 2005
```

Comment:
You can remove that too, it's not _stricto sensus_ part of the legislative text. But good practice to copy the text over from LegiFrance!